src/proxy-animation.js

```diff
@@ -1126,7 +1134,7 @@ export class ProxyAnimation {
       effect: null,
       // The animation attachment range, restricting the animation’s
       // active interval to that range of a timeline
-      animationRange: isScrollAnimation ? parseAnimationRange(timeline, animOptions['animation-range']) : null,
+      animationRange: isScrollAnimation ? parseAnimationRange(animOptions['animation-range']) : null,
```

Comment:
As I understand it, the syntax allows for both `<length-percentage>` and `<timeline-range-name> <length-percentage>?` which makes sense since `animation-range` is a property of the animation and not the timeline. 

The values are checked when used.

When `<length-percentage>` is defined on a view-timeline, the offset corresponds to a specified point on the view-timeline. `animation-range: 0% 100%` is then treated similarly to `cover 0% cover 100%` and `normal`. 

> [`<length-percentage>`](https://drafts.csswg.org/css-values-4/#typedef-length-percentage)
The [animation attachment range](https://drafts.csswg.org/scroll-animations-1/#animation-attachment-range) starts at the specified point on the [timeline](https://drafts.csswg.org/web-animations-1/#timeline) measuring from the start of the timeline.

When `<timeline-range-name> <length-percentage>?` is used on a scroll-timeline, only the offsets are used. `entry 0% entry 100%` is treated similarly to `0% 100%` and `normal`.

This seem to correspond well with Chrome, as far as I have tested. Specify `animation-range: cover 20%` on an animation with a scroll-timeline, and rangeStart will return `{rangeName: 'cover', offset: CSS.px(20)}`. 

I don't see how we can validate a specified `cover 20%` for a ScrollTimeline, as the timeline might be changed to a ViewTimeline afterwards.